go-controller/pkg/cluster/node.go

```diff
@@ -60,7 +60,7 @@ func isOVNControllerReady(name string) (bool, error) {
 	err = wait.PollImmediate(500*time.Millisecond, 60*time.Second, func() (bool, error) {
 		stdout, _, err := util.RunOVSOfctl("dump-aggregate", "br-int")
 		if err != nil {
-			return false, fmt.Errorf("failed to get aggregate flow statistics: %v", err)
+			return false, nil
```

Comment:
@dcbw this one should return error.

If we have already ensured that  OVS bridge `br-int` exists @L49, then `ovs-ofctl dump-aggregate br-int` should definitely succeed. If not, then we have a major issue and return error immediately instead of retrying and returning the error after 30 seconds.

Reply:
ah ok. I'll drop this then.